docs/rst/manual/getting_started/configuration/poly_new/guide.rst

```diff
@@ -0,0 +1,291 @@
+Creating a config
+=================
+Ert runs a set of simulations with prior set of parameters sampled from distributions. The results of the simulations are then given to an optimization algorithm which uses a set of observations to tune the parameters in order to make a next run of the simulations produce results that fit better with the set of observations.
```

Comment:
Think this sentence is a bit confusing, should maybe be expanded a bit or at least split in two.